lib/auth0/api/v2/actions.rb

```diff
@@ -0,0 +1,201 @@
+          path = "#{actions_path}/triggers"
+          get(path)
+        end
+
+        # Get an action by id.
+        # @see https://auth0.com/docs/api/management/v2/#!/actions/get_action
+        # @param action_id [string] The action_id of the user to retrieve.
+        #
+        # @return [json] Returns the action with the given action_id if it exists.
+        def action(action_id)
+          raise Auth0::MissingActionId, 'Must supply a valid action_id' if action_id.to_s.empty?
+          path = "#{actions_path}/actions/#{action_id}"
+          get(path)
+        end
+        alias get_action action
+
+
+        # Deletes a single action given its id
+        # @see https://auth0.com/docs/api/management/v2/#!/actions/delete_action
+        # @param action_id [string] The action ID
+        # @param force [boolean] Force action deletion detaching bindings (defaults to false)
+        def delete_action(action_id, force=false)
+          raise Auth0::MissingActionId, 'Must supply a valid action_id' if action_id.to_s.empty?
+          path = "#{actions_path}/actions/#{action_id}"
+          delete(path, { force: force })
+        end
+
+        # Update an existing action.
+        # @see https://auth0.com/docs/api/management/v2/#!/Actions/patch_action
+        # @param action_id [string] The action ID
+        # @param body [hash] The optional parameters to update.
+        #
+        # @return [json] Returns the updated user.
+        def patch_action(action_id, body)
+          raise Auth0::MissingActionId, 'Must supply a valid action_id' if action_id.to_s.empty?
+          raise Auth0::InvalidParameter, 'Must supply a valid body' if body.to_s.empty? || body.empty?
+          path = "#{actions_path}/actions/#{action_id}"
+          patch(path, body)
+        end
+        alias update_action patch_action
+
+        # Retrieve information about a specific execution of a trigger.
+        # @see https://auth0.com/docs/api/management/v2/#!/actions/get_action
+        # @param execution_id [string] The ID of the exeution to retrieve.
+        #
+        # @return [json] Returns the action with the given execution_id if it exists.
+        def execution(execution_id)
+          raise Auth0::MissingExecutionId, 'Must supply a valid execution_id' if execution_id.to_s.empty?
+          path = "#{actions_path}/executions/#{execution_id}"
+          get(path)
+        end
+        alias get_execution execution
+
+        # Retrieve the actions that are bound to a trigger.
+        # @see https://auth0.com/docs/api/management/v2/#!/actions/get_bindings
+        # @param trigger_id [string] An actions extensibility point.
+        # @param per_page [integer] The amount of entries per page. Default: 50. Max value: 100.
+        # @param page [integer] The page number. Zero based
+        #
+        # @return [json] Returns the action with the given trigger_id if it exists.
+        def trigger_bindings(trigger_id, page: nil, per_page: nil)
```

Comment:
Currently the endpoint does not support the `include_totals` parameter, but it will likely be added. Will it be easy to add support for that parameter in a non-breaking way here?

Reply:
yes, named parameters are basically destructuring a hash argument